libraries/api-documenter/src/index.ts

```diff
@@ -2,12 +2,30 @@
 // See LICENSE in the project root for license information.
 
 import * as colors from 'colors';
+import * as fsx from 'fs-extra';
 import * as os from 'os';
 import * as path from 'path';
+import { Documenter } from './Documenter';
+import { MarkdownPageRenderer } from './MarkdownPageRenderer';
 
 const myPackageJsonFilename: string = path.resolve(path.join(
   __dirname, '..', 'package.json')
 );
 const myPackageJson: { version: string } = require(myPackageJsonFilename);
 
 console.log(os.EOL + colors.bold(`api-documenter ${myPackageJson.version}` + os.EOL));
+
+const documenter: Documenter = new Documenter();
+const inputFolder: string = path.join(__dirname, '../files/input');
```

Comment:
>path.join(__dirname, '../files/input'); [](start = 28, length = 39)

what is this path?

Reply:
Currently it reads its inputs/outputs from a fixed folder location.  In a future PR I will add a proper command-line interface

---
In reply to: [135166886](https://github.com/Microsoft/web-build-tools/pull/322#discussion_r135166886) [](ancestors = 135166886)